Update dev ui start workflow page (#226)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc

image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-workflow-definitions-filter.png[]

[[con-trigger-cloud-event-page]]
== Start New Workflow page

> The Start New Workflow page is used start a workflow instance through a cloud event or a custom form depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of workflow definitions table.

Reviewer: ```suggestion
The Start New Workflow page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of the workflow definitions table.
```

> The Start New Workflow page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click on the play button in the *Actions* column of the workflow definitions table.

Reviewer 2: ```suggestion
The *Start New Workflow* page is used to start a workflow instance through a cloud event or a custom form, depending on the workflow configuration. To navigate to the *Start New Workflow* page, click the play button in the *Actions* column of the workflow definitions table.
```

> If there is no json schema for the workflow, then the workflow will be started by triggering a cloud event.

Reviewer: ```suggestion
*If there is no json schema for the workflow*, then the workflow will be started by triggering a cloud event.
```

> *If there is no json schema for the workflow*, then the workflow will be started by triggering a cloud event.

Reviewer 2: ```suggestion
*If there is no JSON schema for the workflow, then the workflow is started by triggering a cloud event.
```

> To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively, and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.

Reviewer 2: ```suggestion
To trigger a cloud event, you can use the *Cloud Event Type* and *Cloud Event Data* input fields to set the type of a cloud event and event payload in JSON format respectively and click the *Start* button to start a workflow. The *Reset* button on the page resets the values entered in the given fields.
```

> .Starting a workflow using cloud event

Reviewer 2: ```suggestion
.Starting a workflow using a cloud event
```

image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events.png[]

> If the workflow has a json schema configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.

Reviewer: ```suggestion
*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.
```

> *If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in-order to start a new workflow instance. You can fill-in the required form details and click on the *Start* button to trigger the workflow.

Reviewer 3: ```suggestion
*If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in order to start a new workflow instance. You can fill in the required form details and click on the *Start* button to trigger the workflow.
```

> *If there is a json schema for the workflow* configured in the `dataInputSchema` property, then a form is displayed in order to start a new workflow instance. You can fill in the required form details and click on the *Start* button to trigger the workflow.

Reviewer 2: ```suggestion
*If there is a JSON schema for the workflow configured in the `dataInputSchema` property, then a form is displayed to start a new workflow instance. You can fill in the required form details and click the *Start* button to trigger the workflow.
```

The *Reset* button is used to clear the form data.

> .Starting a workflow using form

Reviewer 2: ```suggestion
.Starting a workflow using the form
```

image::tooling/quarkus-dev-ui-extension/kogito-swf-tools-trigger-cloud-events-custom-form.png[]

> You can also use the *Business key* text box to define a custom business key value to the workflow instance. If the *Business Key* field is blank, then an auto-generated business key is defined to the workflow instance.

Reviewer 2: ```suggestion
You can also use the *Business key* text box to define a custom business key value to the workflow instance. If the *Business Key* field is blank, then an auto-generated business key is defined for the workflow instance.
```
